config/algolia_settings.json

```diff
@@ -96,14 +96,7 @@
     ],
     "exactOnSingleWordQuery": "attribute",
     "ranking": [
-      "typo",
-      "geo",
-      "words",
-      "filters",
-      "proximity",
-      "attribute",
-      "exact",
-      "custom"
+      "desc(publication_date_timestamp)"
```

Comment:
This is from an export I just did after Ben changed the config - does anyone know what happened here? 😕